digdag-standards/src/main/java/io/digdag/standards/command/DockerCommandExecutor.java

```diff
@@ -60,23 +60,28 @@ public Process start(Path projectPath, TaskRequest request, ProcessBuilder pb)
     private Process startWithDocker(Path projectPath, TaskRequest request, ProcessBuilder pb)
     {
         Config dockerConfig = request.getConfig().getNestedOrGetEmpty("docker");
+        Config optionConfig = dockerConfig.getNestedOrGetEmpty("options");
         String baseImageName = dockerConfig.get("image", String.class);
+        String dockerCommand = dockerConfig.get("docker", String.class, "docker");
 
         String imageName;
         if (dockerConfig.has("build")) {
             List<String> buildCommands = dockerConfig.getList("build", String.class);
+            List<String> buildOptions = optionConfig.getListOrEmpty("build", String.class);
             imageName = uniqueImageName(request, baseImageName, buildCommands);
-            buildImage(imageName, projectPath, baseImageName, buildCommands);
+            buildImage(dockerCommand, buildOptions, imageName, projectPath, baseImageName, buildCommands);
         }
         else {
             imageName = baseImageName;
             if (dockerConfig.get("pull_always", Boolean.class, false)) {
-                pullImage(imageName);
+                List<String> pullOptions = optionConfig.getListOrEmpty("pull", String.class);
```

Comment:
How can "pull" option be used? If you can share your usecase for this option, please let us know.

Reply:
```
Usage:  docker pull [OPTIONS] NAME[:TAG|@DIGEST]

Pull an image or a repository from a registry

Options:
  -a, --all-tags                Download all tagged images in the repository
      --disable-content-trust   Skip image verification (default true)
```

docker pull has only two options, -a and --disable-content-trust.
Users may use `pull: ["--disable-content-trust"]`.

Reply:
But, even I probably do not use the "pull" options. It is okay to remove it.

Reply:
I decided to remove `pull` options for now to keep it simple.

Reply:
ok, I agree with your decision.